services/MenusService.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * MenusService
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Eyal Wolanowski <eyal.wolanowski@gmail.com>
+ * @copyright Copyright (c) 2018 Eyal Wolanowski <eyal.wolanowski@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+
+namespace OpenEMR\Services;
+
+use OpenEMR\Menu\PatientMenuRole;
+
+class MenusService
+{
+
+  /**
+   * Default constructor.
+   */
+    public function __construct()
+    {
+    }
+
+
+    public function getMenu($pid)
+    {
+
+        $menuPatient = new PatientMenuRole();
+        $menu_restrictions=$menuPatient->getMenu();
+        $cat_id = 1;
```

Comment:
Someone coming here after the fact is going to wonder what in the world is this cat_id 1 and option_id 1, provide comments here please.